ruby/red-arrow/test/test-slicer.rb

```diff
@@ -484,4 +484,38 @@ def setup
 7	   256	true   
     TABLE
   end
+
+  sub_test_case "MatchSubstringOptions family" do
+    def setup
+      @string_table = Arrow::Table.new(
+        index: [*1..5],
+        string: ["array", "Arrow", "carrot", nil, "window"]
+      )
+    end
+
+    test("match_substring") do
+      sliced_table = @string_table.slice do |slicer|
+        slicer.string.match_substring("arr")
+      end
+      assert_equal(<<~TABLE, sliced_table.to_s)
+	 index	string
+0	     1	array 
+1	     3	carrot
+2	(null)	(null)
+      TABLE
+    end
+
+    test("match_substring with ignore_case option") do
```

Comment:
Could you use simple Ruby syntax like test name instead of English?
I like the style because it's easy to read for me :-)

```suggestion
    test("match_substring(ignore_case:)") do
```